netlify.toml

```diff
@@ -80,6 +80,10 @@
   from = "/base/colors"
   to = "/product/base/colors"
 
+[[redirects]]
```

Comment:
@dancormier I don't think this redirect rule is necessary, this is a brand new page. In fact I do think we should check in netlify if those rules are actually still executed (e.g. there is still some references somewhere in the world pointing to them) otherwise we should consider removing them. 🙂

Reply:
Yeah, that was me being heavy-handed with duplicating _everything_ we have for the legacy colors. I'll get rid of this change at the very least.